packages/commons/src/events/utils.ts

```diff
@@ -251,3 +254,35 @@ export function stripHiddenFields(fields: FieldConfig[], data: EventState) {
     return !isFieldVisible(field, data)
   })
 }
+
+export function findActiveDrafts(event: EventDocument, drafts: Draft[]) {
+  const actions = event.actions
+    .slice()
+    .sort((a, b) => a.createdAt.localeCompare(b.createdAt))
+
+  const lastAction = actions[actions.length - 1]
+  return drafts
+    .filter(({ createdAt }) => createdAt >= lastAction.createdAt)
```

Comment:
```suggestion
    .filter(({ createdAt }) => createdAt > lastAction.createdAt)
```

This should of course be like this but temporally allows equal timestamps as the generated demo data is not perfect yet